enlighten/parser/ColumnFileParser.py

```diff
@@ -83,10 +83,20 @@ def parse(self) -> Measurement:
     # ##########################################################################
 
     def get_safe_float(self, field):
```

Comment:
this method should be in Common.py and take the dictionary as an argument

Reply:
I'm hoping we can leave it local, now that it's smaller and simpler (less worthy of re-use)